apps/storybook.namekit.io/stories/Namekit/TextArea.stories.tsx

```diff
@@ -92,11 +91,21 @@ export const CustomClass: Story = {
   },
 };
 
-export const Disabled: Story = {
+export const PrimaryDisabled: Story = {
   args: {
     variant: "primary",
     size: "medium",
-    value: "Disabled Textarea",
+    value: "Primary Disabled Textarea",
```

Comment:
Please see the issues I reported about using `value` like this.

Reply:
@lightwalker-eth Done, I've changed that to use defaultValue instead of value